Sprint-2/implement/querystring.test.js

```diff
@@ -5,8 +5,30 @@
 
 const parseQueryString = require("./querystring.js")
 
-test("parses querystring values containing =", () => {
-  expect(parseQueryString("equation=x=y+1")).toEqual({
-    "equation": "x=y+1",
+describe("parseQueryString()", () => {
+  test("parses querystring values containing multiple =", () => {
+    expect(parseQueryString("equation=x=y+1")).toEqual({equation: 'x'});
```

Comment:
The expected returned value for this test case should be:
`expect(parseQueryString("equation=x=y+1")).toEqual({equation: 'x=y+1'});`